bindata/cloud-network-config-controller/admission-controller.yaml

```diff
@@ -0,0 +1,69 @@
+---
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: cloud-network-config-admission-controller
+  namespace: openshift-cloud-network-config-controller
+  labels:
+    app: cloud-network-config-admission-controller
+  annotations:
+    kubernetes.io/description: |
+      This daemon set launches the cloud network config admission controller component on each master node.
+    release.openshift.io/version: "{{.ReleaseVersion}}"
+    networkoperator.openshift.io/non-critical: ""
```

Comment:
`networkoperator.openshift.io/non-critical: ""` is a CNO specific annotation which just tells the CNO to ignore this resource when computing the status. The cloud-network-config-controller is not host networked (because it doesn't need to be) and will moreover depend on the cloud-credentials-operator for the cloud credentials it requires to talk to the cloud API. These components will not start until the network is up and running, but if the CNO doesn't skip these components it will never be ready and the cluster deployment will never reach the point where the CCO is deployed.